functions/src/venue.js

```diff
@@ -1,3 +1,5 @@
+// TODO: Rename this file to venue.ts (or venue/index.ts) + refactor to make proper use of TypeScript
```

Comment:
[TODO] Rename this file to `venue.ts` (or `venue/index.ts`) + refactor to make proper use of TypeScript